shell/browser/native_window.h

```diff
@@ -159,6 +159,7 @@ class NativeWindow : public base::SupportsUserData,
   virtual bool IsTabletMode() const;
   virtual void SetBackgroundColor(SkColor color) = 0;
   virtual SkColor GetBackgroundColor() = 0;
+  virtual void InvalidateShadow() = 0;
```

Comment:
The code fails to compile on Linux/Windows do to lack of implementation. Maybe wrap it in `#if defined(OS_MAC)`?